script/Arcueid/Day 9/09_00_ARC09_4.txt

```diff
@@ -166,12 +180,12 @@ The sound of cicadas.
 [sha:1b88ddfc5adb8385c2ea350180a33b38f7ca09ec]{
 -- Page 132, Offset 21651.
 -- 意識が焼かれる。
-My consciousness heats up.
+My consciousness is being burned to a crisp.
 }
 [sha:c3b19e0e20faf5758ad48f198c9d40f72d67b696]{
 -- Page 132, Offset 21652.
 -- どすん、と熱砂の地面に膝をつく。
-With a thud, I fall to my knees, onto the scorching sand.
+With a thud, I fall to my knees on the scorching sand.
```

Comment:
is there sand in the CG? I can't find any pictures of what 熱砂 is supposed to be because its apparently the name of some BL game... I wonder, is it literal?

Reply:
It didn't _look_ sandy, but you mostly just see grass. I can go back and check. Weblio doesn't really have any other meanings listed for this one though.

Reply:
/shrug
![2022-03-26-194727_1920x1140_scrot](https://user-images.githubusercontent.com/2476799/160260766-bf319b9e-eed2-403f-82e3-e9aa955efbf2.png)

Reply:
_maybe_ it's an allusion to the dry, brown grass? Feels like a bit of a stretch though.

Reply:
meh, its what it says. I just was a little bit confused because I (apparently correctly) didn't remember any sand